Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.py

```diff
@@ -0,0 +1,322 @@
+from CommonServerPython import *  # noqa: E402
+import demistomock as demisto  # noqa: E402
+from AWSApiModule import *
+from typing import TYPE_CHECKING
+
+# The following imports are used only for type hints and autocomplete.
+# They are not used at runtime, and are not in the docker image.
+if TYPE_CHECKING:
+    from mypy_boto3_organizations import *
+    from botocore.paginate import Paginator
```

Comment:
why are we using imports that arent available in the image? Better practice just not to use these at all